memiavl/snapshot.go

```diff
@@ -20,30 +23,44 @@ const (
 	// the initial snapshot format
 	SnapshotFormat = 0
 
-	// magic: uint32, format: uint32, version: uint64, root node index: uint32
-	SizeMetadata = 20
+	// magic: uint32, format: uint32, version: uint32, root node index: uint32
+	SizeMetadata = 16
 
 	// EmptyRootNodeIndex is a special value of root node index to represent empty tree
 	EmptyRootNodeIndex = math.MaxUint32
+
+	Alignment = 8
+
+	OffsetRestartInteval = 65536
+
```

Comment:
```suggestion
	Alignment = 8

	OffsetRestartInteval = 65536

```

Reply:
cleaned up, they are not used anymore.

Reply:
maybe can clean RoaringBitmap related deps as well